app/src/main/res/layout/message_tile.xml

```diff
@@ -57,7 +57,7 @@
             android:paddingLeft="8dp"
             android:paddingRight="16dp"
             android:paddingStart="8dp"
-            tools:text="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam mollis pretium purus, faucibus accumsan enim placerat non. " />
+            tools:text="@string/template" />
```

Comment:
Rename the strings which are used as placeholders with a suffix `_placeholder ` it would help while translating!